src/extensions/editable/bootstrap-table-editable.js

```diff
@@ -34,7 +42,7 @@
             return;
         }
 
-        $.each(this.columns, function (i, column) {
+        $.each(this.options.columns, function (i, column) {
```

Comment:
In our latest version, because our plugin support `rowspan/colspan` table header, so we have to use `this.columns` to replace `this.options.columns`.